CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Want to hack on the Moby Project? Awesome! We have a contributor's guide that ex
 [setting up a development environment and the contribution
 process](docs/contributing/). 
 
-[![Contributors guide](docs/static_files/contributors.png)](https://docs.docker.com/opensource/project/who-written-for/)
+[![Contributors guide](docs/static_files/contributors.png)](https://docs.docker.com/contribute/)
```

Comment:
The https://docs.docker.com/contribute/ is purely for contributing to the docs. The right URL probably should be https://github.com/moby/moby/blob/master/docs/contributing/who-written-for.md

The image can be found in this repository as well, but wondering if we should still include it, because the image itself refers to "docs.docker.com"; 
![](https://github.com/moby/moby/blob/master/docs/static_files/contributors.png)